### ISSUE TITLE
Allow map viewing without downloading whole areas

### ISSUE THREAD

**Issue by troxler** [mapsme/omim#4461]
Sometimes it would be very useful to just browse the map when you are online. Right now, the app forces you to download a whole area first before you can further zoom in. Would it be possible to make the force-download optional when you are online? The app should just download the data that is needed on-demand (like Google Maps).

**Comment by burivuh** [comment]
No, it is against our strategy to focus on offline side of every feature.

**Comment by troxler** [comment]
I love the offline functionality and I don't think this feature is *against* that strategy. It just makes maps.me more versatile as it works great offline *and* online.

**Comment by burivuh** [comment]
It is actually against our _focus_. We lack manpower, so we can't do anything outside our strategy scope.
